instances/species/susScrofaDomesticus.jsonld

```diff
@@ -0,0 +1,19 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/species/susScrofaDomesticus",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Species",
+  "definition": "The species 'Sus scrofa domesticus' (domestic pig) is a mammal of the artiodactyla (even-toed ungulates) order and genus sus (wild and domestic pigs).",
```

Comment:
"The species *Sus scrofa domesticus* (domestic pig) belongs to the family of *suidae* (suids)."

Reply:
fixed